geonode/qgis_server/helpers.py

```diff
@@ -82,3 +89,138 @@ def validate_django_settings():
             "{package}.".format(package=qgis_server.BACKEND_PACKAGE))
 
     return True
+
+
+def map_thumbnail_url(instance):
```

Comment:
Ok, I guess it's much better as it will be integrated with Geonode resources (set thumbnail manually etc, list of resources ...)